js/src/widgets/editorPanel.js

```diff
@@ -92,6 +97,7 @@
             jQuery.subscribe('openAnnotationList.' + this.windowId, function(event, data) {
                 _this.openAnnotationList(data);
             });
+
```

Comment:
We might want to create a new events listener here for the page turn in order to clear or invalidate the rendering.